homework/04LessonFour.js

```diff
@@ -76,6 +92,8 @@ var nestedObjects = {
     }
   }
 };
+
+nestedObjects['users']['bob roberts']['starring'] = 'Tim Robbins';
```

Comment:
Great use of bracket notation here!

![dancingmarshmallow](https://media.giphy.com/media/l46CdUTy21h8wwrAs/giphy.gif)